### ISSUE TITLE
%run with single-quoted path doesn't work in IPython 5.1

### ISSUE THREAD

**Issue by scottcode** [ipython/ipython#10079]
Using `%run '<script_path>'` to run another script (single-quoted) fails in IPython 5.1, whereas it used to work in IPython 4.2. Using double-quotes still works, as well as not using quotes at all. 

# My setup

* Windows 7
* IPython 5.1
* Python 2.7

Output of `python -c "import IPython; print(IPython.sys_info())"`:

```
{'commit_hash': u'5c9c918',
 'commit_source': 'installation',
 'default_encoding': 'cp437',
 'ipython_path': 'C:\\Users\\myusername\\AppData\\Local\\Continuum\\Anaconda2\\lib\\site-packages\\IPython',
 'ipython_version': '5.1.0',
 'os_name': 'nt',
 'platform': 'Windows-7-6.1.7601-SP1',
 'sys_executable': 'C:\\Users\\myusername\\AppData\\Local\\Continuum\\Anaconda2\\python.exe',
 'sys_platform': 'win32',
 'sys_version': '2.7.12 |Anaconda 4.1.1 (64-bit)| (default, Jun 29 2016, 11:07:13) [MSC v.1500 64 bit (AMD64)]'}
```

# Test case

1. Copy this snippet into a file called "dummy.py":
```
x = range(100)
```

2. Open IPython console and type:
```
%run 'dummy.py'
```
3. I get ``ERROR:root:File `u"'dummy.py'.py"` not found.``
4. Try `%run "dummy.py"` or `%run dummy.py` instead --> no error.
5. The `x` variable from "dummy.py" can now be used within the IPython shell (if double- or no-quotes was used)

```
In [3]:  print x[:5]
Out[3]: [0, 1, 2, 3, 4]
```

**Comment by anntzer** [comment]
It's very likely.  The problem is that single quotes are valid characters in the Windows console, and the normal way to escape filenames in Windows is always to use double quotes (which are *not* valid filenames).  I don't think coming up with our own escaping mechanism is really better (e.g., backslashes used to be a problem as they are separators on Windows).

**Comment by takluyver** [comment]
Does using single quotes work in cmd/Powershell? E.g. can you run `python 'foo.py'? If that doesn't work, I'd say it's OK for it not to work in IPython either.

**Comment by anntzer** [comment]
Single quotes do not work in cmd.
From some quick testing they *do* appear to work in powershell, but there the proper way to escaped single quotes within the string is to double them (`foo'bar` -> `"foo'bar"` or `'foo''bar'`).  (See also https://msdn.microsoft.com/en-us/powershell/reference/5.0/microsoft.powershell.core/about/about_quoting_rules.)

For simplicity I'd suggest sticking to the cmd rules...

**Comment by ivanov** [comment]
If the prevailing opinion is to emulate cmd behavior and not support single quotes on windows, since `cmd` does not do that, either, then I think it would be a useful contribution for someone to document this, so I am tagging this issue for new contributors to find it as such.

**Comment by kd2718** [comment]
I just added one line. I didn't build docs to make sure it was in the correct place. Will test more later

**Comment by kd2718** [comment]
I think this is ready. I had some issues building docs on my machine, but the changes look like they took.

There is a note in the docs/README.rst file about seeing fully built docs on travis. However, I wasn't sure where to look to actually see the fully built docs in the travis interface.

> Documentation builds are included in the Travis-CI continuous integration process, so you can see the results of the docs build for any pull request

**Comment by takluyver** [comment]
Closed by #11377.

The README note is somewhat cocnfusing. You can see the terminal output (e.g. warnings) resulting from building docs on Travis, but as far as I know you can't see the built docs themselves. PRs to improve the README are also welcome.

**Comment by koushikVss** [comment]
why its so shitty just to run a python script using ipython.......I tried %run demopy
